python/scraper.py

```diff
@@ -23,12 +23,22 @@ def log(msg, level=xbmc.LOGDEBUG):
 def get_tmdb_scraper(settings):
     language = settings.getSettingString('language')
     certcountry = settings.getSettingString('tmdbcertcountry')
-    return TMDBMovieScraper(ADDON_SETTINGS, language, certcountry)
+    search_language = settings.getSettingString('searchlanguage')
```

Comment:
croneter is correct, this change also need a new setting in [the settings.xml file](https://github.com/xbmc/metadata.themoviedb.org.python/blob/master/resources/settings.xml#L8). You can copy this line and change `label="30002"` attribute value to your new language string **30013** and the `id="language"` attribute value to **searchlanguage** to match this line.